### PR TITLE
Implement Prisma

### PR DESCRIPTION
**Pull request by dylhack**
Test Checklist - utilize Prisma studio to double-check that each table is storing data properly.

 - User
   - [ ] can edit messages
   - [ ] can delete messages
   - [ ] can send messages
   - [ ] can open thread
   - [ ] can send attachment
   - [ ] can send multiple attachments 
   - [ ] can select category
 - Staff
   - [ ] can edit messages
   - [ ] can delete messages
   - [ ] can send messages
   - [ ] can forward thread to same category origin
   - [ ] can forward thread to cross origin category
   - [ ] can close thread
   - [ ] can open thread
   - [ ] can send attachment
   - [ ] can send multiple attachments
   - [ ] can view messages in admin panel

### CODE REVIEW

**Comment by dylhack** [comment]
waiting for: #15

**Comment by dylhack** [comment]
This is not ready for review until it is out of draft

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+npm exec prisma generate
+npm exec tsc
+
```
Comment (AltF02) [inline scripts/build.sh:4]: use package.json scripts

**Review by AltF02 — CHANGES_REQUESTED** [review]

**Comment by dylhack** [comment]
@AltF02 Could you add more tests to the todo list?

**Pull request closed without merging**